### PR TITLE
chore(ci): update Karma to 0.12.32-beta.0

### PR DESCRIPTION
**Pull request by juliemr**
This will hopefully make the CI more stable because of its updated
version of socket.io.

### CODE REVIEW

**Comment by juliemr** [comment]
NOTE - this is a work in progress pointing to github instead of npm. I will update to point to 0.12.32-beta.0 once that is released, if this passes.

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "jasmine-node": "~1.14.5",
     "jasmine-reporters": "~1.0.1",
     "jshint-stylish": "~1.0.0",
-    "karma": "~0.12.0",
+    "karma": "^0.12.32-beta.0",
```
Comment (IgorMinar) [inline package.json]: do we want to use `^`? that will update us automatically to 0.13/0.14 when that is available and somebody does `npm update`. Sounds too risky to me.

**Comment by IgorMinar** [comment]
otherwise lgtm

File: package.json (hunk 1 above)
Comment (juliemr) [inline package.json]: Done

**Comment by juliemr** [comment]
I had to rename the karma release to 0.12.32 because otherwise peer dependencies would fail and npm wouldn't install.

**Comment by juliemr** [comment]
Merged to master as 581ee9d0b6aacd76d9ede56ab6c735b2691f9193

**Comment by juliemr** [comment]
Merged to v1.3.x as 01a725a7697181dbcc92853c18f486430c26e625

**Pull request closed without merging**